src/templates/Studio/IntroBlock/Block/Block.js

```diff
@@ -44,7 +44,7 @@ const Block = ({ image, index, copy, title, video }) => {
           className={styles.mediaWrapper}
           style={{ scale }}
         >
-          {video ? (
+          {/* {video ? (
```

Comment:
Can be removed later on - in case it becomes clear we will not support video at all